### PR TITLE
PCHR-2030: Admin Dashboard HTML+CSS

### PR DESCRIPTION
**Pull request by deb1990**
## Overview
As part of this PR the Static HTML Markup and CSS has been developed.

## Technical Details
1. `leaveandabsence.css` has been added to `Dashboard.php` as it holds all the styles for Leave and Absence.
2. `_admin-dashboard-page.scss` is created under `uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/` folder, and it holds the custom styles for the Admin Dashboard Page.
3. Static HTML has been put into `Dashboard.tpl` which needs to be moved to `admin-dashboard-container.html` as part of PCHR-2031.
4. All the styles for `_admin-dashboard-page.scss` moved to `@root`, because after moving `@import "pages/*";` inside `#bootstrap-theme` namespace, the styles were not working as both `page-civicrm-leaveandabsences-dashboard` and `admin-dashboard-page` class is outside of the namespace.

## Comment
New variant of outline button i.e btn-default-outline has been added to the Shoreditch theme.
https://github.com/compucorp/org.civicrm.shoreditch/pull/21
This PR should be merged only after the above PR is merged and 1.7 is synced.

## Refactored Leave Request Popup HTML's
**Reason**: Previously the HTMLs were not properly organized, 
1. They were in different folders
request-ctrl_js_-_hr17-1_-____buildkit_build_hr17-1_

2. `leave-request-popup.html` file was too large to handle. 

**Solution**: 
1. Header and Footer portion of `leave-request-popup.html` has been moved to `leave-request-popup-header.html` and `leave-request-popup-footer.html` respectively.

2. Markup for each tab(details, comments and files) has been moved to `leave-request-popup-details-tab.html` , `leave-request-popup-comments-tab.html` and `leave-request-popup-files-tab.html` respectively.

3. All the content of `request-modal` folder has been moved to `leave-request-popup/details-tab/` folder and `leave.html` has been renamed to `common.html` which makes more sence as this markup is used for all 3 leave types.

**Current Folder Structure**
leave-request-popup_html_-_hr17-1_-____buildkit_build_hr17-1_

**Future Refactoring Plans**
Next we need to create sub components for details, comments and files tabs as the Javascript code is also too large in `request-ctrl`. The already created HTML partials for each tab need to be used.

### CODE REVIEW

File: uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+    display: none !important;
+  }
+
+  #page {
+    margin: 0;
+    padding: 0;
+  }
+}
+
+.admin-dashboard-page {
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss]: all the styling contained here seem overrides of the Shoreditch style. Why are they necessary?

File: uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.page-civicrm-leaveandabsences-dashboard {
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss]: What is the style inside this selector for?

File: uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/Dashboard.tpl [hunk 3]
```diff
@@ -1 +1,22 @@
...
+      </button>
+      <ul class="dropdown-menu" role="menu">
+        <li><a href="#"><i class="fa fa-clock-o" aria-hidden="true"></i> Record Staff Late</a></li>
+        <li><a href="#"><i class="fa fa-calendar-plus-o" aria-hidden="true"></i> Record Staff Overtime</a></li>
+        <li><a href="#"><i class="fa fa-stethoscope" aria-hidden="true"></i> Record Staff Sickness</a></li>
+        <li><a href="#"><i class="fa fa-briefcase" aria-hidden="true"></i> Record Staff Leave</a></li>
+      </ul>
+    </div>
+  </div>
+
```
Comment (AkA84) [inline uk.co.compucorp.civicrm.hrleaveandabsences/templates/CRM/HRLeaveAndAbsences/Page/Dashboard.tpl]: empty line

File: uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss (hunk 1 above)
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss]: 1. `panel-heading` - Shoreditch has white color for heading background, here the color is different.
2. `dropdown-toggle`- Shoreditch does not have a button with transparent background and white font color.
3. `chr_admin-dashboard_side-nav  li.active`- Shoreditch does not have the variant of Tab/Nav that we need. The color of background for selected tab, is different here.

All the above are in respect to styleguide.

File: uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss (hunk 2 above)
Comment (deb1990) [inline uk.co.compucorp.civicrm.hrleaveandabsences/scss/pages/_admin-dashboard-page.scss]: dashboard___hr17-1

**Pull request merged**